examples/service/notifications_consumer.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from miteD.service.service import rpc_service, notification_handler
+
+
+@rpc_service(
+    name='notifications_consumer',
+    versions=['1.0'],
+    broker_urls=['nats://127.0.0.1:4222'],
+)
+class NotificationsConsumer:
+    """
+    To define notifications handler use 'notification_handler' decorator as in examples below.
+    """
+
+    """
+    Listen on channel "notification.service.notifications_producer.1_0.updates"
+    """
+    @notification_handler(
+        layer='service',
+        name='notifications_producer',
+        versions=['1.0'],
+        topic='updates',
+    )
+    async def updates_handler(self, channel, msg):
+        logging.debug("updates_handler: got notification from channel: {} msg = '{}'".format(channel, msg))
+        raise RuntimeError('dddup')
+
+    """
+    Listen on channel "notification.service.notifications_producer.1_0.errors"
+    """
+    @notification_handler(
+        layer='service',
+        name='notifications_producer',
+        versions=['1.0'],
```

Comment:
so versions here would mean that we only get notifications from the specified versions of those services? What would be a good example of this in use, just asking to understand better

Reply:
Yes, if you specify version, it would only handle events from that specific version. Otherwise default value of '*' is used.

I didn't have any specific usage example on my mind, it's just a matter of consistency - if we support versioning on api/rpc calls we should also support it for notifications mechanism.
We can use it or not, the mechanism is there and can be used if there's a need for that.